openapi/schemas/admin/TaskResponse.yaml

```diff
@@ -0,0 +1,9 @@
+type: "object"
+required: ["taskId", "name"]
+properties:
+  taskId:
+    type: "integer"
+    format: "int64"
+  name:
+    type: "string"
+    description: "The name of the task"
```

Comment:
It might also require some changes in the engine, but maybe we should add the missing `description` field that tasks also have